Assets/Game/ScriptableObjects/Spawn by Shortcut.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Is this file needed? the file name seems create by accident

Reply:
Liked many meta files, this file might just modified by unity by itself when I open or close the game on Unity.